docutils.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, lib, fetchPypi, buildPythonPackage, isPy3k, python }:
```

Comment:
nixpkgs contains `docutils` 0.18. Is there a reason to package docutils 0.17? Instead could you bump `nixpkgs`?

Reply:
The RTD theme doesn't work with newer docutils, hence the anchor on `docutils-0.17`

Reply:
Ha, great.